client/me/concierge/cancel/index.js

```diff
@@ -29,37 +29,64 @@ class ConciergeCancel extends Component {
 		siteSlug: PropTypes.string.isRequired,
 	};
 
+	cancelAppointment = () => {
+		const { appointmentId } = this.props;
+		this.props.cancelConciergeAppointment( WPCOM_CONCIERGE_SCHEDULE_ID, appointmentId );
+	};
+
 	getDisplayComponent = () => {
-		const { siteSlug, signupForm, translate } = this.props;
+		const { appointmentId, siteSlug, signupForm, translate } = this.props;
 
 		switch ( signupForm.status ) {
 			case CONCIERGE_STATUS_CANCELLED:
 				return (
 					<Confirmation
-						buttonLabel={ translate( 'Schedule' ) }
-						buttonUrl={ `/me/concierge/${ siteSlug }/book` }
 						description={ translate( 'Would you like to schedule a new session?' ) }
 						title={ translate( 'Your Concierge session has been cancelled.' ) }
-					/>
+					>
+						<Button
+							className="cancel__schedule-button"
+							href={ `/me/concierge/${ siteSlug }/book` }
+							primary={ true }
+						>
+							{ translate( 'Schedule' ) }
+						</Button>
+					</Confirmation>
 				);
 
-			case CONCIERGE_STATUS_CANCELLING_ERROR:
+			default:
 				return (
-					<Card highlight="error">
-						{ translate( "We couldn't cancel your session, please try again later." ) }
-					</Card>
-				);
+					<Confirmation
+						description={ translate(
+							'You can also reschedule your session. What would you like to do?'
+						) }
+						title={ translate( 'Cancel your Concierge session' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Cancelling your Concierge session…' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/concierge-cancelling-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).